app/templates/components/recirculation-prefooter.hbs

```diff
@@ -47,6 +26,26 @@
   </div>
 {{/if}}
 
+{{#if displaySponsoredContent}}
```

Comment:
is it ok for sponsored content to be below wiki articles?

Reply:
according to designs it should be in this place, see in the IW-1469 ticket